.vsts-ci.yml

```diff
@@ -217,7 +217,8 @@ stages:
               _LinuxPortable: ''
               _RuntimeIdentifier: '--runtime-id linux-musl-x64'
               _BuildArchitecture: 'x64'
-              _AdditionalBuildParameters: '/p:OSName="linux-musl"'
+              # Pass in HostOSName when running on alpine
```

Comment:
would be good to validate this on all build platforms particularly macOS since that isnt in the default PR CI.

Reply:
I tested on an Mac M1 and the build works, but required manually codesigning crossgen2 executable. Dont believe installer would be building on an M1 though so it would appear MacOS x64 should be working.

Reply:
just realized Darwin is the MacOS variant. So resolving this.